.github/workflows/sdlc-sdk-update.yml

```diff
@@ -0,0 +1,165 @@
+name: SDLC / SDK Update
+run-name: "SDK ${{inputs.run-mode == 'Update' && format('Update - {0}', inputs.sdk-version) || format('Test #{0} - {1}', inputs.pr-id, inputs.sdk-version)}}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-mode:
+        description: "Run Mode"
+        type: choice
+        options:
+          - Test # used for testing sdk-internal repo PRs
+          - Update # opens a PR in this repo updating the SDK
+        default: Test
+      #
+      sdk-package:
+        description: "SDK Package ID"
+        required: true
+        default: "BitwardenSdk"
+      sdk-version:
+        description: "SDK Version"
+        required: true
+        default: "2a6609428275c758fcda5383bfb6b3166ec29eda"
```

Comment:
🤔 By having a default, couldn't it misfire and go back to this revision instead of updating to the one in fact needed? IMO this should be `required` with no default so every time this flow is executed a revision must be passed.